simulation/agents.py

```diff
@@ -102,5 +107,28 @@ def renovation_budget(self) -> float:
             self.wealth_percentile,
         )
 
+    def evaluate_renovation(self, model) -> None:
+
+        step_interval_years = model.step_interval / datetime.timedelta(days=365)
+        proba_renovate = model.annual_renovation_rate * step_interval_years
+
+        self.is_renovating = True if random.random() < proba_renovate else False
```

Comment:
We will have a lot of the `True if random.random() < proba_renovate else False` statements. Note that this is just the same as:
```suggestion
        self.is_renovating = (random.random() < proba_renovate)
```
and we should replace all instances in the code with that. 

I'm wondering if there is a simpler `@staticmethod` we can add to the agent since we will use this a lot. I'm struggling to think of a good, concise name, though, but something like `true_with_probability(p: float)`

what do you think of:
```
@staticmethod
def true_with_probability(p: float) -> bool:
    return random.random() < p
```

Reply:
I'm also happy for this not to be a staticmethod, but just to be a function defined above the agents itself. That way, we will save a lot of `self.true_with_probability` stuff, and I think this method is general enough it can be defined above.